### PR TITLE
test: add dynamic root nodes and interpolation

### PR DESCRIPTION
**Pull request by baiwusanyu-c**
~~Handled errors in some use cases.~~
Added unit test about dynamic root nodes and interpolation
```
<template>
  <button @click="handleClick" :id="count">
   {{count}}
  </button>
</template>
```

```
<template>
  <button @click="handleClick" :id="count">
    foo {{count}}
  </button>
</template>
```

```
<template>
  <button @click="handleClick" :id="count">
    foo {{count}} foo
  </button>
</template>
```

### CODE REVIEW

**Comment by autofix-troubleshooter** [comment]
Hi! I'm the <a href="https://autofix.ci">autofix logo<strong>autofix.ci</strong></a> troubleshooter bot.

It looks like you correctly set up a CI job that uses the autofix.ci GitHub Action, but the [**autofix.ci GitHub App**](https://github.com/apps/autofix-ci/) has not been installed for this repository. This means that autofix.ci unfortunately does not have the permissions to fix this pull request. If you are the repository owner, please install the app and then restart the CI workflow! 😃

File: packages/compiler-vapor/__tests__/basic.test.ts [hunk 1]
```diff
@@ -2,7 +2,10 @@ import * as CompilerVapor from '../src'
 // import * as CompilerDOM from '@vue/compiler-dom'
 import { parse, compileScript } from '@vue/compiler-sfc'
 import source from './fixtures/counter.vue?raw'
-
+import singleRootSource from './fixtures/singleRoot.vue?raw'
```
Comment (sxzz) [inline packages/compiler-vapor/__tests__/basic.test.ts:5]: The fixtures test is for dev only. Please add unit test separately.
See https://github.com/vuejs/core-vapor/blob/acec2409c7694c5dd37a3f173fe432899549866e/packages/compiler-vapor/__tests__/compile.test.ts

**Review by sxzz — CHANGES_REQUESTED** [review]
This problem is resolved via 91e7f9bb835936f8b0f8edf5750c9bedeb0cf8c5

But the unit tested is still needed

File: packages/compiler-vapor/__tests__/compile.test.ts [hunk 2]
```diff
@@ -33,6 +33,13 @@ describe('comile', () => {
     expect(code).matchSnapshot()
   })
 
+  test('Dynamic root nodes and interpolation', async () => {
```
Comment (sxzz) [inline packages/compiler-vapor/__tests__/compile.test.ts:36]:
```suggestion
  test('dynamic root nodes and interpolation', async () => {
```

**Review by sxzz — COMMENTED** [review]
Please remove empty file `packages/compiler-vapor/__tests__/__snapshots__/basic.test.ts.snap`

**Review by sxzz — APPROVED** [review]

**Pull request merged**